crates/fuel-core/src/database.rs

```diff
@@ -76,50 +77,60 @@ pub mod vm_database;
 )]
 pub enum Column {
     /// The column id of metadata about the blockchain
-    Metadata = 0,
+    Metadata,
```

Comment:
one advantage of assigning integers here, is that we can remove columns in the future without causing a lot of problems on existing datasets.

Reply:
We can handle it later in the `as_usize` function, if we need this functionality

Reply:
Honestly don't love this as it makes the enum order dependent